src/demo/demo.ejs

```diff
@@ -19,29 +19,28 @@
     ">
     <% } %>
     <style type="text/css">
-      html, body, .container {
-        height: 100%;
-        margin: 0;
-      }
-      body {
-        display: flex;
-        align-items: center;
-        justify-content: center;
-      }
-      #onfido-mount {
+      html, body {
         width: 100%;
         height: 100%;
-        display: flex;
-        align-items: center;
-        justify-content: center;
+        margin: 0;
+        padding: 0;
       }
       #demo-app {
         position: relative;
```

Comment:
I think you want to remove this double `position: relative` here

Reply:
It causes the SDK to disappear when you squeeze the window

Reply:
Good spot! 😱 
Have fixed it and removed the duplicated `position: relative`